docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,148 @@
+# Release 405 (28 Nov 2022)
+
+## General
+
+* Improve performance of queries that compare `date` columns with
+  `timestamp with time zone` literals. ({issue}`5798`)
+* Improve performance and resource utilization of `INSERT` queries into
+  partitioned and bucketed tables. ({issue}`14718`)
+* Fix overflows of `TIME` and `TIME WITH TIME ZONE` types by rounding to zero. ({issue}`15138`)
+* Fix incorrect query results when an un-partitioned `WINDOW` operator is
```

Comment:
"window operator" is not a term most users will understand. I would rephrase in terms of what users interact with:

> Fix incorrect results for queries involving window functions without a `PARTITION BY` clause followed by the evaluation of window functions with a `PARTITON BY` and `ORDER BY` clause.